### PR TITLE
3301 add ghg units to org model

### PR DESCRIPTION
**Pull request by perryr16**
#### Any background context you want to provide?
SEED columns with data types of 'area' and 'eui' were appended with a unit in the inventory detail and inventory list, e.g. "Gross Floor Area (ft²)". GHG and GHG Intensity columns were displayed without units, despite having hardcoded unit values on the PropertyState model.

#### What's this PR do?
- Updates the organization model to recognize `ghg` and `ghg_intensity` units 
- Allows users to select GHG and intensity units on the mapping page. 
- Adds GHG and intensity units to column headers on the inventory list and detail

Note: Due to Pints pretty formatting the following units are converted for printing `kgCO2e --> kg` and `MtCO2e --> t` 

#### How should this be manually tested?
- Upload a property file with GHG and GHG intensity columns
- confirm the mapping screen has a dropdown of the correct unit type (currently only one available)
- confirm GHG and intensity column headers have correct units appended on the inventory list and display

#### What are the relevant tickets?
#3301 

#### Screenshots (if appropriate)

Screen Shot 2022-07-21 at 3 58 34 PM
Screen Shot 2022-07-21 at 3 56 37 PM
Screen Shot 2022-07-21 at 3 56 10 PM

### CODE REVIEW

File: seed/lib/superperms/orgs/models.py [hunk 1]
```diff
@@ -123,6 +123,14 @@ class Organization(models.Model):
         ('kBtu/m**2/year', 'kBtu/m²/year'),  # really, Toronto?
     )
 
+    MEASUREMENT_CHOICES_GHG = (
+        ('MtCO2e/year', 'MtCO2e/year'),
+    )
+
+    MEASUREMENT_CHOICES_GHG_INTENSITY = (
+        ('kgCO2e/ft**2/year', 'kgCO2e/ft²/year'),
+    )
```
Comment (perryr16) [inline seed/lib/superperms/orgs/models.py:132]: Not sure of other units, but the foundation is set so changes should be easy

**Review by haneslinger — APPROVED** [review]
🥇

File: seed/static/seed/js/controllers/mapping_controller.js [hunk 2]
```diff
@@ -302,12 +302,26 @@ angular.module('BE.seed.controller.mapping', [])
...
+      var ghg_columns = _.filter($scope.mappable_property_columns, {data_type: 'ghg'});
+      $scope.is_ghg_column = function (col) {
+        return col.suggestion_table_name == 'PropertyState' && Boolean(_.find(ghg_columns, {column_name: col.suggestion_column_name}))
+      }
+
+      var ghg_intensity_columns = _.filter($scope.mappable_property_columns, {data_type: 'ghg_intensity'});
+      $scope.is_ghg_intensity_column = function (col) {
+        return col.suggestion_table_name == 'PropertyState' && Boolean(_.find(ghg_intensity_columns, {column_name: col.suggestion_column_name}))
+      }
+
```
Comment (Ryoken) [inline seed/static/seed/js/controllers/mapping_controller.js:314]: Looks good, but can this file get semicolons on 307, 312, 322 and 234?

**Review by Ryoken — APPROVED** [review]
Looks good (minus a couple semicolons)!

**Pull request merged**